.github/workflows/CI.yml

```diff
@@ -13,8 +13,6 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '0.7'
-          - '1.0'
```

Comment:
It's dangerous to stop testing 1.0 while declaring that we also support it. We could either change the [compat] to `julia = "1.6"` or go back to testing 1.0. I lean slightly towards dropping support for 1.0, but I'm fine with either one.

Reply:
This was another case where I just didn't notice it, fixed.